spec/commands/binstubs_spec.rb

```diff
@@ -100,14 +100,22 @@
         bundle! "binstubs bundler rack prints_loaded_gems"
       end
 
-      let(:system_bundler_version) { Bundler::VERSION }
+      let(:system_bundler_version) { Gem::Version.new(Bundler::VERSION).bump.to_s }
```

Comment:
Could you add a comment here why the bump is necessary?

Reply:
It makes sense. I added its detail at https://github.com/bundler/bundler/pull/6291/commits/cf5dd0e6a35ddfc554f705099dfba207c3b54583